app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -208,6 +209,9 @@ class BrowserFragment : Fragment(), BackHandler {
             view = view
         )
 
+        val actionSheet = view.findViewById<QuickActionSheet>(R.id.quick_action_sheet)
+        actionSheet.afterInflate()
```

Comment:
I know this is not in line with how we have the other `integration` and `feature`s in this file, but creating an integration for this seemed a little overkill for just this visual component. If we decide to move forward with this UX we could go about making the integration, but feel free to let me know if it's necessary now!

Reply:
One could override OnAttachedToWindow() instead. Does it matter if we've called onDraw yet?

Reply:
Ah, that's exactly what I was looking for. Thank you! 😄